app/core/application.coffee

```diff
@@ -61,6 +60,16 @@ Application = {
     )
     store.commit('me/updateUser', me.attributes)
     store.commit('updateFeatures', features)
+    if me.showChinaRemindToast()
+      setInterval ( -> noty {
+        text: '你已经练习了一个小时了，建议休息一会儿哦'
```

Comment:
Do you think we should place the toast text into locale file or somewhere else? 
@nwinter @AndrewJakubowicz @codeluggage